### ISSUE TITLE
Server Crash when applying redstone to Wireless Transmitter block

### ISSUE THREAD

**Issue by psychofad** [raoulvdberge/refinedstorage#338]
#### Steps to reproduce:
1.  Setup a wireless transmitter in the overworld connected to your main network and set the redstone setting to active without a redstone signal
2. Connect a Wireless redstone receiver from RFTools to the block and set up a wireless redstone transmitter in an alternate dimension facing a block with a button on it
3.  Press the button to take down the server and delete all of your items for you ( :P )
...

#### Version (Make sure you are on the latest version before reporting):

- Minecraft: 1.10.2
- Forge: forge-1.10.2-12.18.1.2086
- Refined Storage: refinedstorage-1.0.1

Does this issue occur on a server? [yes/no] YES

#### If a (crash)log is relevant for this issue, link it here:

[http://pastebin.com/npS28n6J](http://pastebin.com/npS28n6J)

**Comment by psychofad** [comment]
I just tried to reproduce this after moving some things around (there were 3 of the transmitters being reset at once) and the block reset and there was no crash.  Odd.  Anyways, seems to be working for now, but you have the crash log from before in case you want to add in checks to prevent this type of crash.

**Comment by raoulvdberge** [comment]
Please send me all the storage mediums you are using. For external storages, also state to what type of inventory its connected

**Comment by psychofad** [comment]
The only storage mediums hooked up was the one disk drive with 8 of the 64k Storage disks in it.  The entire network consists of the following:

Main Network:
1 Network Controller
1 Disk Drive (with 8 64k Disks)
1 Crafting Grid
1 Pattern Grid
1 Import bus connected to an Ender Link chest from MultiStorage mod
1 Export Bus connected to an Ender Link chest from MultiStorage mod
2 Crafters (Still have the old patterns in them that need to be remade)
3 Network Transmitters (1 with the interdimensional upgrade)
1 Solderer 

Satellite Network 1 (Wireless Connection 1 - Overworld):
1 Network Receiver
1 Crafting Grid

Satellite Network 2 (Wireless Connection 2 - Overworld)
1 Network Receiver
8 Crafters (4 connected to Sag Mills and 4 Connected to Alloy Smelters from EnderIO)
9 Export Buses (4 connected to Sag Mills and 4 Connected to Alloy Smelters from EnderIO, and 1 connected to a chest that supplies silk touched ores to a block placer)
17 Import Buses (8 connected to Sag Mills and 8 Connected to Alloy Smelters from EnderIO, and 1 connected to a chest that is connected to a block breaker)

Interdimensional Network (Wireless Connection 3 - Mining World from Utility Worlds Mod)
1 Network Receiver
1 Crafting Grid

That is all I have hooked up right now, but I have plans to expand the one in the mining world to include some farms and other things.  Hope this helps in some way.

**Comment by psychofad** [comment]
Just thought of something else, when I crashed I also had an export bus going into an Automatic Ore Converter from the Ore Dictionary Converter mod and an import bus pulling the items back out of it, both had stack upgrades in them and I had temporarily connected the cables up to an accessible crafter block to convert some ingots from one type to another.   I had almost forgotten about this since I had to restore a backup that was made before I had done this.

I have tried to reproduce the crash on my test server with the old setup, including the ore converter, and I can not get it to crash again.  This is the strangest thing.
